### DISCUSSION TITLE
如何解决正在下载缺失的 geoip.dat 和 geosite.dat，请稍后刷新页面。

### DISCUSSION THREAD

**Discussion by hulyoo** [v2rayA/v2rayA#1417] [Q&A]
Downloading missing geoip.dat and geosite.dat; refresh the page later.
正在下载缺失的 geoip.dat 和 geosite.dat，请稍后刷新页面。
过一会刷新界面，界面就丢失了，v2raya.service也failed了。这该怎么解决啊

**Comment by Gaojianli** [comment]
请提供日志

**Reply by hulyoo** [reply]
重新安装一边已经好了
